community/examples/Gallery.shtml

```diff
@@ -78,19 +79,19 @@ He's created a very nice series of web pages on
 <a href="http://www.quaker-valley.com/CTC/QV_CTCnew.html">how it's done</a>.
 <p>
 
-<h3>Alan Huberty's Bloomfield &amp; Oakland RR</h3>
+<h3>Bloomfield &amp; Oakland RR</h3>
 
 Alan Huberty created a multi-screen control panel for his Bloomfield &amp; Oakland Railroad.
 <a href="images/BORRCTCPanel.gif"><img src="images/BORRCTCPanel.gif" width="512" height="192"></a>
 
-<h3>Otis McGee's SP Shasta Route</h3>
```

Comment:
Sorry, just now getting back to this.  Is there a reason behind removing the layout owner's name here? (and also Alan Huberty's name)  They were both OK with this originally.  There are a _lot_ of Shasta Route model railroads (three in this area alone)